private/protocol/json/jsonutil/build.go

```diff
@@ -25,6 +25,7 @@ func BuildJSON(v interface{}) ([]byte, error) {
 }
 
 func buildAny(value reflect.Value, buf *bytes.Buffer, tag reflect.StructTag) error {
+	v := value
```

Comment:
Suggest something like `origVal` or similar so its clearer the difference between `value` and `v`.